packages/arb-util/configuration/configuration.go

```diff
@@ -0,0 +1,393 @@
+	Port    string        `koanf:"port"`
+	Ping    time.Duration `koanf:"ping"`
+	Timeout time.Duration `koanf:"timeout"`
+	Queue   int           `koanf:"queue"`
+	Workers int           `koanf:"workers"`
+}
+
+type Feed struct {
+	Input  FeedInput  `koanf:"input"`
+	Output FeedOutput `koanf:"output"`
+}
+
+type Healthcheck struct {
+	Addr   string `koanf:"addr"`
+	Enable bool   `koanf:"enable"`
+	L1Node struct {
+		Enable bool `koanf:"enable"`
+	} `koanf:"l1-node"`
+	Metrics struct {
+		Enable bool   `koanf:"enable"`
+		Prefix string `koanf:"prefix"`
+	} `koanf:"metrics"`
+	Port      string `koanf:"port"`
+	Sequencer struct {
+		Enable bool `koanf:"enable"`
+	} `koanf:"sequencer"`
+}
+
+type Node struct {
+	Aggregator struct {
+		Inbox struct {
+			Address string `koanf:"address"`
+		} `koanf:"inbox"`
+		MaxBatchTime int64 `koanf:"max-batch-time"`
+		Stateful     bool  `koanf:"stateful"`
+	} `koanf:"aggregator"`
+	Forward struct {
+		URL string `koanf:"url"`
+	} `koanf:"forward"`
+	Sequencer struct {
+		CreateBatchBlockInterval   int64 `koanf:"create-batch-block-interval"`
+		DelayedMessagesTargetDelay int64 `koanf:"delayed-messages-target-delay"`
+		Enable                     bool  `koanf:"enable"`
+	} `koanf:"sequencer"`
+}
+
+type Persistent struct {
+	Database struct {
+		Path string `koanf:"path"`
+	} `koanf:"database"`
+	Storage struct {
+		Path string `koanf:"path"`
+	} `koanf:"storage"`
+}
+
+type Rollup struct {
+	Address   string `koanf:"address"`
+	ChainID   uint64 `koanf:"chain-id"`
+	FromBlock int64  `koanf:"from-block"`
+	Machine   struct {
+		Filename string `koanf:"filename"`
+		//URL string `koanf:"url"`
+	} `koanf:"machine"`
+}
+
+type Validator struct {
+	Strategy string `koanf:"strategy"`
+	Utils    struct {
+		Address string `koanf:"address"`
+	} `koanf:"utils"`
+	WalletFactory struct {
+		Address string `koanf:"address"`
+	} `koanf:"wallet-factory"`
+}
+
+type Wallet struct {
+	Password string  `koanf:"password"`
+	GasPrice float64 `koanf:"gas-price"`
+}
+
+type Config struct {
+	Bridge Bridge `koanf:"bridge"`
+	Conf   string `koanf:"conf"`
+	Dump   struct {
+		Conf bool `koanf:"conf"`
+	} `koanf:"dump"`
+	Feed        Feed        `koanf:"feed"`
+	GasPriceUrl string      `koanf:"gas-price-url"`
+	Healthcheck Healthcheck `koanf:"healthcheck"`
+	L1          struct {
+		URL string `koanf:"url"`
+	} `koanf:"l1"`
+	Log struct {
+		RPC  string `koanf:"rpc"`
+		Core string `koanf:"core"`
+	} `koanf:"log"`
+	Node       Node       `kaonf:"sequencer"`
+	Persistent Persistent `koanf:"persistent"`
+	PProf      struct {
+		Enable bool `koanf:"enable"`
+	} `koanf:"pprof"`
+	Rollup Rollup `koanf:"rollup"`
+	RPC    struct {
+		Addr string `koanf:"addr"`
+		Port string `koanf:"port"`
+	} `koanf:"rpc"`
+	Validator     Validator `koanf:"validator"`
+	WaitToCatchUp bool      `koanf:"wait-to-catch-up"`
+	Wallet        Wallet    `koanf:"wallet"`
+	WS            struct {
+		Addr string `koanf:"addr"`
+		Port string `koanf:"port"`
+	} `koanf:"ws"`
```

Comment:
Probably should move RPC and WS into Node

Reply:
sounds good